core/backend/src/domains/FunctionalSchema.ts

```diff
@@ -27,7 +27,7 @@ export class FunctionalSchema extends Schema {
     }
   }
 
-  /** @deprecated in 2.x. Use [[schemaFilePath]] and IModelDb.importSchemas instead */
```

Comment:
@ColinKerr @calebmshafer I don't understand why this method was deprecated. The deprecation message was useless to me. If the function `nativeDb.importFunctionalSchema` should be removed, then someone should do that and replace the implementation of this function with the correct code.

We shouldn't be using a deprecated-in-2.x function in 4.0.